src/openslide-vendor-dicom.c

```diff
@@ -278,6 +279,21 @@ static bool get_tag_str(const DcmDataSet *dataset,
          dcm_element_get_value_string(NULL, element, index, result);
 }
 
+static bool get_tag_binary(const DcmDataSet *dataset,
+                           const char *keyword,
+                           const char **result,
```

Comment:
`const void **` since the result is a binary blob?

Reply:
OK, changed.